utils/utils.go

```diff
@@ -75,6 +75,13 @@ func PackIPAsUint32(ip net.IP) uint32 {
 	return 0
 }
 
+// UnpackUint32AsIP does the reverse of PackIPAsUint32
+func UnpackUint32AsIP(ip uint32) net.IP {
```

Comment:
please move new functions to internal/utils.go. we don't want to be adding more public methods that are not part of the client API.